src/descriptive.js

```diff
@@ -28,10 +28,31 @@ export function max(arr) {
   return result;
 }
 
+/**
+ * Return the sum of all entries in a numeric array.
+ * @param {Number[]} arr the data array
+ * @returns {Number} the sum of all entries in the array
+ */
+export function sum(arr) {
+  if (!Array.isArray(arr)) return undefined;
+  if (arr.length === 0) return 0;
```

Comment:
As stated in the doc, this should return undefined.
```suggestion
  if (!Array.isArray(arr) || arr.length === 0) return undefined;
```